package.json

```diff
@@ -15,6 +15,7 @@
     "@react-navigation/bottom-tabs": "^6.3.1",
     "@react-navigation/native": "^6.0.10",
     "@react-navigation/native-stack": "^6.6.2",
+    "@types/react-native": "^0.67.7",
```

Comment:
https://stackoverflow.com/questions/71535491/how-to-prevent-vs-code-auto-importing-from-react-native-web